examples/tutorials/advanced/date_time_charts.py

```diff
@@ -191,10 +191,10 @@
 #
 # In this example, instead of using a list of :class:`pandas.DatetimeIndex` objects,
 # ``x`` is initialized as an :class:`xarray.DataArray` object. This object provides a
-# wrapper around regular PyData formats. It also allows the data to have labeled
-# dimensions while supporting operations that use various pieces of metadata. The
-# following code uses :func:`pandas.date_range` to fill the DataArray with data, but
-# this is not essential for the creation of a valid DataArray.
+# wrapper around regular scientific Python formats. It also allows the data to have
```

Comment:
> This object provides a wrapper around regular scientific Python formats.

I'm unsure what "scientific Python formats" means. The official documentation (https://docs.xarray.dev/en/stable/generated/xarray.DataArray.html) says:

> DataArray provides a wrapper around numpy ndarrays that uses labeled dimensions and coordinates to support metadata aware operations.

Reply:
Good point!. Actually I was already a bit confused by "regular PyData formats". So, I just replaced the term. Maybe we should rewrite this sentence to something similar as in the official xarray docs, please see commit https://github.com/GenericMappingTools/pygmt/pull/3447/commits/27735247fb2a09dc3b58a4c0ef55ff2e7e4461fa.